README.md

```diff
@@ -212,6 +212,63 @@ From here on you can follow the instructions available at the following sections
 
 All done!
 
+Install ansible-openwisp2 on docker container for development
```

Comment:
I would change this to:

`How to run tests`.

Reply:
:+1: